app/models/event.rb

```diff
@@ -27,6 +27,7 @@ class Event < ActiveRecord::Base
   friendly_id :name, use: :slugged
 
   scope :upcoming_events, -> { where("end_date >= ?", Time.now) }
+  scope :past_events, -> { where("start_date < ?", Time.now) }
```

Comment:
This might conflict with the upcoming_events scope. Would you please change it to look to the `end_date` attribute instead?